UnicornRender/video/include/unicorn/video/Renderer.hpp

```diff
@@ -68,30 +69,21 @@ class Renderer
      */
     UNICORN_EXPORT virtual void SetDepthTest(bool enabled) = 0;
 
-    /** @brief  Creates new geometry mesh
-    *
-    *  Creates and subscribes to mesh.
-    *  Mesh shall be deleted via DeleteMesh().
-    *
-    *  @param [in] material describes mesh visual representation
-    *
-    *  @attention  Mesh lifetime is bound by its renderer's lifetime.
-    *              Using meshes after their renderer was destroyed is undefined behaviour.
-    *              If you're storing mesh pointers, consider storing them with a reference
-    *              to their renderer and listening for its Destroyed event for proper cleanup.
-    *
-    *  @return pointer to newly created geometry::Mesh
-    *
-    *  @sa DeleteMesh
+    /**
+    * @brief Adds mesh to the rendering system
+    * @param [in] mesh mesh data
+    * @return true if mesh was successfully added to the system
     */
-    UNICORN_EXPORT virtual Mesh* SpawnMesh(Material const& material) = 0;
+    UNICORN_EXPORT virtual bool AddMesh(Mesh* mesh) = 0;
 
     /**
-    * @brief Removes mesh from rendering system
-    * @param [in] mesh pointer to mesh
-    * @return true if mesh was sucessfully deleted from system and false if not
+    * @brief Removes internal rendering mesh data from rendering system
+    *
```

Comment:
Please add explicit `@note` or `@attention` that `pMesh` is not invalidated within the method (if it's not, I'm not quite sure because of indirections, could you check please?)